sdk/search/azure-search-documents/README.md

```diff
@@ -113,61 +168,258 @@ SearchAsyncClient searchAsyncClient = new SearchClientBuilder()
     .buildAsyncClient();
 ```
 
-## Key concepts
+### Send your first search query
 
-Azure Cognitive Search service has the concepts of search services and indexes and documents, where a search service contains 
-one or more indexes that provides persistent storage of searchable data, and data is loaded in the form of JSON documents. 
-Data can be pushed to an index from an external data source, but if you use an indexer, it's possible to crawl a data 
-source to extract and load data into an index.
+To get running immediately, we're going to connect to a well-known sandbox
+Search service provided by Microsoft. This means you do not need an Azure
+subscription or Azure Cognitive Search service to try out this query.
 
-There are several types of operations that can be executed against the service:
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L122-L147 -->
+```Java
+// We'll connect to the Azure Cognitive Search public sandbox and send a
+// query to its "nycjobs" index built from a public dataset of available jobs
+// in New York.
+String serviceName = "azs-playground";
+String indexName = "nycjobs";
+String apiKey = "252044BE3886FE4A8E3BAA4F595114BB";
+
+// Create a SearchClient to send queries
+String serviceEndpoint = String.format("https://%s.search.windows.net/", serviceName);
+AzureKeyCredential credential = new AzureKeyCredential(apiKey);
+SearchClient client = new SearchClientBuilder()
+    .endpoint(serviceEndpoint)
+    .credential(credential)
+    .indexName(indexName)
+    .buildClient();
 
--   [Index management operations](https://docs.microsoft.com/en-us/rest/api/searchservice/index-operations). Create, delete, update, or configure a search index.
--   [Document operations](https://docs.microsoft.com/en-us/rest/api/searchservice/document-operations). Add, update, or delete documents in the index, query the index, or look up specific documents by ID.
--   [Indexer operations](https://docs.microsoft.com/en-us/rest/api/searchservice/indexer-operations). Automate aspects of an indexing operation by configuring a data source and an indexer that you can schedule or run on demand. This feature is supported for a limited number of data source types.
--   [Skillset operations](https://docs.microsoft.com/en-us/rest/api/searchservice/skillset-operations). Part of AI workload, a skillset defines a series of a series of enrichment processing steps. A skillset is consumed by an indexer.
--   [Synonym map operations](https://docs.microsoft.com/en-us/rest/api/searchservice/synonym-map-operations). A synonym map is a service-level resource that contains user-defined synonyms. This resource is maintained independently from search indexes. Once uploaded, you can point any searchable field to the synonym map (one per field).
+// Let's get the top 5 jobs related to Microsoft
+SearchPagedIterable searchResultsIterable = client.search("Microsoft", new SearchOptions().setTop(5),
+    Context.NONE);
```

Comment:
Is there a reason we want to show an example with the Context overload?
usually readme snippets show the easier use-cases?

Reply:
This is one of the champion scenarios to search the top 5 results which has to use maximum overload.